### ISSUE TITLE
popups not showing when using atom-typescript

### ISSUE THREAD

**Issue by jon49** [carlosdcastillo/vim-mode#32]
The popups/intellisense aren't not showing when using atom-typescript. And the files only compile when the plugin is saved and synced with a javascript file before starting to use the neovim plugin.

**Comment by carlosdcastillo** [comment]
Yeah, I could believe that. A variety of plugins could easily not work. neovim-mode can a possessive beast. Can you post a screen shot? How can I easily install atom-typescript?

**Comment by jon49** [comment]
I won't much time until the weekend but I'll get you the screen shot then if you don't mind. So, I could do it then. I don't there is much to atom-typescript install wise. I'm pretty new to atom, so hopefully I didn't mess anything up.

https://atom.io/packages/atom-typescript
